### PR TITLE
Refactor changelog

### PR DESCRIPTION
**Pull request by Linkid**

### CODE REVIEW

**Comment by hramezani** [comment]
Where did you extract these changes logs?

**Comment by Linkid** [comment]
Changes come from the [release notes page](https://github.com/jazzband/django-auditlog/releases) and from the comparison of tags with no notes.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
+# Changes
+
+## 1.0.0 (unreleased)
+
+### Final (unreleased)
```
Comment (hramezani) [inline CHANGELOG.md:5]: What is the meaning of `Final` here?

**Review by hramezani — APPROVED** [review]

**Comment by hramezani** [comment]
Great job @Linkid :+1:

**Pull request merged**